SENDER_DASHBOARD_STRUCTURE.md

```
@@ -0,0 +1,246 @@
+# SenderDashboard Component Structure
+
+## Component Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)